programs/drift/src/instructions/keeper.rs

```diff
@@ -518,6 +518,7 @@ pub fn handle_liquidate_perp(
         slot,
         now,
         state,
+        state.initial_pct_to_liquidate as u128,
```

Comment:
kinda awk that we pass state for just this one (because `update_amm` needs state) but that can be a future refactor